packages/block-editor/src/store/defaults.js

```diff
@@ -131,5 +133,8 @@ export const SETTINGS_DEFAULTS = {
 
 	// List of allowed mime types and file extensions.
 	allowedMimeTypes: null,
+
+	availableLegacyWidgets: {},
```

Comment:
I think if we had GET /widgets we would not need this. But I wanted to avoid creating this endpoint without knowing yet what endpoint needs we will have. Probably after it may make sense to refactor and have this information exposed as a widgets endpoint.